### ISSUE TITLE
Where can you get Bluesuite now?

### ISSUE THREAD

**Issue by witnessmenow** [lorf/csr-spi-ftdi#30]
Seems like bluesuite downloads has been removed from the CSR website, at least for me anyways.

Is there an alternative place to download it?

**Comment by lorf** [comment]
Yeah, they withdrawn all access from my account too. You, probably, need an old good search.

**Comment by witnessmenow** [comment]
Thanks for the reply,

I actually still have the files but I have a guide on how to update the HC-05/06 and I was wondering was there an alternative place I could link to.

**Comment by ridvanaltun** [comment]
Bluesuite 2.5: https://drive.google.com/file/d/1LYr8Q3gmp2ZNQ1yTXH8BocoMLCbXSwA2/view?usp=sharing

Bluesuite 2.6.6: https://drive.google.com/file/d/1d0VRoN2BFZpIo8pR75E049reoKHjuba6/view?usp=sharing

I hope these links are helpful to anyone.

**Comment by kennyjudo** [comment]
Thanks for sharing, by the way, I have a CSR8510 A10 USB dongle, how can I command the dongle? where can I have the command list? I would like to use python for a small project

**Comment by fantom3l** [comment]
For those in need, that like me have searched for BlueSuite, i have uploaded a few versions that i found around , even the latest at this point BlueSuite 2.6.11 build 1937
here: https://yadi.sk/d/QAdQ7zuP-X62Xg

**Comment by el-samiyel** [comment]
Hi all, does anybody know where I can get bluesuite version 2.1?

**Comment by Road-Drum** [comment]
I'm kinda being picky, but is there any known build Bluesuite for Linux? I feel more comfortable on programming this thing on Linux but if only Windows builds are available, I'll just live with it.

and thanks @fantom3l for the latest version available

**Comment by rklosinski** [comment]
Does anyone know how to get the software development kit for the csr8675?

**Comment by Alverzilin** [comment]
Hello friends! I really need version 2.6.2! help me please

**Comment by RayoSP** [comment]
http://www.cisdo.cn/post/5.html

**Comment by ssashton** [comment]
I have Bluesuit 2.6.6. I have successfully used it to change the name on my CSR8675 modules in the past, which used the CSR8675 BGA chip.

I got a new module today that uses the CSR8675 WLCSP chip (smaller IC, fewer pins) but it only says 'chip not recognised' when I use PSKey or BlueFlash. BlueTest says unable to open SPI comunication.

Does this mean the WLCSP version of the CSR8675 chip is not recognised by BlueSuite 2.6.6, or could this be locked firmware behaviour? I'm not sure how a locked firmware behaves.

![20201024_164414](https://user-images.githubusercontent.com/55240502/97086262-10198200-161a-11eb-9862-72265b4a8699.jpg)

**Comment by P1on3R91** [comment]
Hi to all.
Who know where can download oldest version 2.3, or another lower then 2.4?
It need for BlueCore 2 (BC212015).
Thanks!

**Comment by Matheus-Garbelini** [comment]
Anyone have any version of BlueLab software development kit (SDK)?

**Comment by ssashton** [comment]
> Hi to all.
> Who know where can download oldest version 2.3, or another lower then 2.4?
> It need for BlueCore 2 (BC212015).
> Thanks!

Fantom3l has some stuff, linked above. https://yadi.sk/d/QAdQ7zuP-X62Xg

**Comment by ssashton** [comment]
> I have Bluesuit 2.6.6. I have successfully used it to change the name on my CSR8675 modules in the past, which used the CSR8675 BGA chip.
> 
> I got a new module today that uses the CSR8675 WLCSP chip (smaller IC, fewer pins) but it only says 'chip not recognised' when I use PSKey or BlueFlash. BlueTest says unable to open SPI comunication.
> 
> Does this mean the WLCSP version of the CSR8675 chip is not recognised by BlueSuite 2.6.6, or could this be locked firmware behaviour? I'm not sure how a locked firmware behaves.
> 

I got it figured out. It was simply that this module needed the SPI-PCM pin pulled high to enable SPI mode. Then it was supported by BlueSuite 2.6.6

**Comment by P1on3R91** [comment]
> > Hi to all.
> > Who know where can download oldest version 2.3, or another lower then 2.4?
> > It need for BlueCore 2 (BC212015).
> > Thanks!
> 
> Fantom3l has some stuff, linked above. https://yadi.sk/d/QAdQ7zuP-X62Xg

It hasn't  version 2.3, i already saw it ((

**Comment by CDRXavier** [comment]
Looks like a copy of the 2010 firmware for RN-42 is not recognized by the PSTool (ver 2.5.0.93 release) despite BlueFlash worked quite well.
However, said version of PSTool is compiled for win2K (note the legacy interface)
The 2.6.6_1311 PSTool look a lot more modern. Others are still legacy.

**Comment by ArtTechGuy** [comment]
Guys, i hope here is still someone who are interested in this topic
I got the problem with fleshing up the HC 05 with RN 42 firmware
When i downloaded a rn42.xdv and go to PStool i see this picture: 
![image](https://user-images.githubusercontent.com/104677771/166102640-fd32ad2c-ddfb-4ec8-ab2b-3606ddd224e8.png)
Its just an *Error* 
Even if i try to merge dump from HC 05 i see this:
![image](https://user-images.githubusercontent.com/104677771/166102634-6b81b75b-713e-4e31-a5c6-6a388a05b25c.png)
Why does it happen?

**Comment by frenchiveruti** [comment]
I've created an archive.org backup in case the drive files go down. 

https://archive.org/details/blue-suite-2-6-6-1311_202212

**Comment by breakboyandre** [comment]
@ArtTechGuy same problem here. Did you solve it? Looks like something wrong with bluetooth chip or it's version

**Comment by ArtTechGuy** [comment]
> @ArtTechGuy same problem here. Did you solve it? Looks like something wrong with bluetooth chip or it's version

Yes, i solved it, u need to get proper HC05 board with right chip. There is somewhere on intenet the topic of how to identify "right" chip so it can be flashed with rn42 firm. Ill check it later and send u a link, im not in PC rn

**Comment by breakboyandre** [comment]
@ArtTechGuy did you find the info about how to identify the right hc-05 module to flash?

**Comment by ArtTechGuy** [comment]
> @ArtTechGuy did you find the info about how to identify the right hc-05 module to flash?

https://github.com/lorf/csr-spi-ftdi/issues/25   Yes, found it, thanks for reminding!

**Comment by breakboyandre** [comment]
@ArtTechGuy thank you so much

**Comment by bulentperktas** [comment]
it seems that it is so hard to find any kind of documentation or tools about Bluecore4-External chips. I found that Bluesuite 2.6.12.3289 version on Qualcomm site but they do not let you use without their customer ID.

And their documentation starts from Qualcomm® BlueCore™ 5 Multimedia External chips, There are not neither BC2, BC3 nor BC4.

Does anybody have this version that 2.6.12?

https://www.qualcomm.com/products/technology/bluetooth/bc57e#Software

Thanks any advise,
Bulent

**Comment by MDVelkov** [comment]
> Seems like bluesuite downloads has been removed from the CSR website, at least for me anyways.
> 
> Is there an alternative place to download it?
[
](

https://support.avantree.com/hc/en-us/articles/39984893796633-Firmware-Update-for-DG60L
